src/main/java/com/nutomic/syncthingandroid/syncthing/AppConfigReceiver.java

```diff
@@ -0,0 +1,101 @@
+package com.nutomic.syncthingandroid.syncthing;
+
+import android.app.Notification;
+import android.app.NotificationManager;
+import android.app.PendingIntent;
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Build;
+import android.os.Bundle;
+import android.support.v4.app.NotificationCompat;
+
+import com.nutomic.syncthingandroid.R;
+import com.nutomic.syncthingandroid.activities.MainActivity;
+
+/**
+ * Broadcast-receiver to control and configure SyncThing remotely
+ *
+ * Created by sqrt-1764 on 25.03.16.
+ */
+public class AppConfigReceiver extends BroadcastReceiver {
+    private static final int ID_NOTIFICATION_BACKGROUND_ACTIVE = 2;
+
+    public static final String ACTION_SYNCTHINGSERVIVE = "syncthingservice";
```

Comment:
A thing that is in the back of my head would be the following:
I use Syncthing also for backup of large folders. So it would be nice to be able to disable synchronisation of some shares if I am "on the road" only enabling them when I am at home and charging - or something like this.
Those settings for the folders would fit perfectly into my current pattern. You could then pass a collection of folders to be enabled/disabled in the extras as well as a command to start or stop the service. Or maybe a restartService command might be handy in some cases?

IMHO this would be a perfect place to interface the app ... all external configuration-changes would then come through this receiver ...

If we limit the function to only starting and stopping the service then the Receiver also should be renamed to reflect this "specialized" behaviour. Something like "conrolSyncthingService" ...